### DISCUSSION TITLE
ESP-01S simple switch on GPIO2

### DISCUSSION THREAD

**Discussion by PeterJensen10** [arendst/Tasmota#20670] [Support request]
Can someone help me configure an esp-01s ESP8266 as a simple switch in Tasmota. I will use it together with a motion sensor that sends out a positive signal for 2 seconds when it is activated. There's something I'm missing because I can't see anything happening when I activate GPIO2. esp-01s have tasmota 13.3 firmware. I have configured it as tasmota Generic 18 and switch on GPIO2 I have tried both positive and negative signal on GPIO2. I guess there is something i have missed. Regards Peter
![Skärmklipp](https://github.com/arendst/Tasmota/assets/159059276/84deee54-8659-43da-b430-5b2b6c33bfcd)

**Comment by sfromis** [comment]
You should not expect to see anything until you actually do something with the input.
Potentially useful: https://tasmota.github.io/docs/PIR-Motion-Sensors/
A simple test would also be to config the pin as a counter, then you should see numbers incrementing on the Tasmota front page. Note that I'd expect the board to have a pullup-resistor on gpio 2.

**Comment by PeterJensen10** [comment]
I have solved it. I had connected wrongly, I am sorry for this request. but thank you very much
